applications/portal/templates/statefulset.yaml

```
@@ -72,23 +72,23 @@ spec:
             - name: PROPS_OP_searchActionsCmdMask_11
               value: showDatalinkTable
             - name: PROPS_OP_dataServiceOptions_targetPanelExampleRow1_0
-              value: "40.1, -34.4"
+              value: "53.01, -28.35"
             - name: PROPS_OP_dataServiceOptions_targetPanelExampleRow1_1
-              value: "106.1, -10.45"
+              value: "37.98, 7.0"
             - name: PROPS_OP_dataServiceOptions_targetPanelExampleRow1_2
-              value: "37.83, 6.95"
+              value: "NGC 1494"
             - name: PROPS_OP_dataServiceOptions_targetPanelExampleRow2_0
-              value: "53.2, -28.089 EQ_J2000"
+              value: "-95.04, -25.0 EQ_J2000"
```

```suggestion
              value: "95.04, -25.0 EQ_J2000"
```
This was my mistake, I put a stray negative RA in the slack message.